/as_drive_node:
  ros__parameters:
    as_topic: "/as/status"
    wait_canopen_topic: "/canopen/ok/op_mode"
    mission_topic: "/mission/selected"
    # 1 -> acceleration
    # 2 -> skidpad
    # 3 -> autocross
    # 4 -> trackdrive
    # 5 -> ebs_test
    # 6 -> inspection
    mission_id: 4
    use_mission_sub: true
    orin_on_topic: "/orin/on"
